lib/src/repository/traits/battery_repository.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:yonomi_platform_sdk/src/queries/devices/get_device/query.data.gql.dart';
+import 'package:yonomi_platform_sdk/src/queries/devices/get_devices/query.data.gql.dart';
+import 'package:yonomi_platform_sdk/yonomi-sdk.dart';
+
+class BatteryRepository {
```

Comment:
I'm liking this approach, but maybe we should call them something else besides Repository classes? Repository classes usually encapsulate the logic required to access data sources, e.g. making a network request or making database calls.